File: src/main/scala/otserver4s/database/ConnectionFactory.scala
package otserver4s.database

case class ConexaoBancoDados()
object ConexaoBancoDados {
  val DATABASE_URL = otserver4s.PropriedadeConfiguracoes("db.url")
  def criarConexao: com.j256.ormlite.support.ConnectionSource = new 
    com.j256.ormlite.jdbc.JdbcConnectionSource(DATABASE_URL)
}

File: src/main/kotlin/otserver4kotlin/ConnectionFactory.kt
package otserver4kotlin

class ConexaoBancoDados {
	companion object {
		val DATABASE_URL = PropriedadeConfiguracoes("db.url")
    fun criarConexao(): com.j256.ormlite.support.ConnectionSource {
      val cs = com.j256.ormlite.jdbc.JdbcPooledConnectionSource(DATABASE_URL)
      cs.setMaxConnectionAgeMillis(
		    PropriedadeConfiguracoes(
				  "db.tempo.vida.maxima.conexao.segundos").toInt() * 1000L)
      return cs
    }
	}
}
